Composer/packages/client/src/recoilModel/dispatchers/project.ts

```diff
@@ -343,6 +344,7 @@ export const projectDispatcher = () => {
   });
 
   const reloadProject = async (callbackHelpers: CallbackInterface, response: any) => {
+    callbackHelpers.reset(filePersistenceState(response.data.id));
```

Comment:
Could the loadProjectData do this.  Is there any time when we want to start trying to save stuff while we are loading?

Reply:
I think reload project should tell the filepersistance to not do anything.